CMakeLists.txt

```diff
@@ -126,17 +126,17 @@ function(clr_unknown_arch)
     message(FATAL_ERROR "Only AMD64, ARM and I386 are supported")
 endfunction()
 
-if(INTL_ICU_SH)
-    unset(INTL_ICU_SH CACHE)    # don't cache
-    add_definitions(-DINTL_ICU=1)
-    set(ICU_INTL_ENABLED 1)
+if(NOINTL_ICU_SH)
+    unset(NOINTL_ICU_SH CACHE)    # don't cache
+    set(ICU_INTL_DISABLED 1)
 endif()
 
 if(ICU_INCLUDE_PATH)
     add_definitions(-DHAS_REAL_ICU=1)
     set(ICU_CC_PATH "${ICU_INCLUDE_PATH}/../lib/")
     find_library(ICUUC icuuc PATHS ${ICU_CC_PATH} NO_DEFAULT_PATH)
-    if(ICU_INTL_ENABLED)
```

Comment:
Since these checks become double-negatives, maybe have both ICU_INTL_ENABLED and ICU_INTL_DISABLED defined above for clearer conditions?

Reply:
well, IMHO that would be a bit more error prone.